book.rb

```diff
@@ -0,0 +1,16 @@
+require './rental'
+require './person'
+
+class Book
+  attr_accessor :title, :author
```

Comment:
- [ ] We need a rental to be listed on the setter/getter here because we need read when we want to see who got the book, and also write it so that we read it.

Reply:
done, as advised